tests/InputFiles/CurvedScalarWave/PlaneWaveMinkowski1D.yaml

```diff
@@ -0,0 +1,71 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolvePlaneWaveMinkowski1D
+# Check: parse;execute
```

Comment:
Can you change `execute` to `execute_check_output` and find something in the output you can check? Maybe you can observe a norm and check it is below a threshold. That gives a bit more confidence that things keep working ok than just checking the executable runs without crashing.

Reply:
Ideally I would like to check that the output is apprx the same as the output by the `ScalarWave` executable but I am not sure how I would go by implementing that without storing the data somewhere.

Alternatively, I can just check the constraint violations.

Reply:
yea running `ScalarWave` first seems like a bit of effort. Checking constraint violations sounds good.